scripts/test-runner-lambdas.py

```
@@ -1508,6 +1640,613 @@ def case_the_roster_answer_is_recorded_on_the_instance():
+    for exact in mismatches:
+        result, ec2, _, _ = lease_poll(
+            tags={"RunnerRegistrationProtocol": "ddb-v1"}, runners=[],
+            dynamodb=FakeDynamoDB([registered_item()]), recheck={77: exact})
+        assert still_running(ec2) == ["i-lease"], (exact, still_running(ec2))
+        assert result["held"] == ["i-lease"], (exact, result)
+
+
+def case_a_malformed_registration_row_holds():
+    """A row that does not describe this instance decides nothing."""
+    rows = []
+    bad_id = registered_item(); bad_id["RunnerId"] = {"N": "not-an-integer"}; rows.append(bad_id)
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)